components/NavBar/NavBar.jsx

```diff
@@ -1,52 +1,81 @@
 import React, { useState } from "react";
 import styles from "./NavBar.module.scss";
 import Link from "next/link";
-
+import Alert from "@material-ui/lab/Alert";
 import DrawerToggleButton from "components/SideDrawer/DrawerToggleButton";
+import InfoIcon from "@material-ui/icons/Info";
+import { createMuiTheme, ThemeProvider } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles({
+  myAlert: {
+    backgroundColor: "white",
+  },
+});
+
+//Colour of banner is not updated
+const theme = createMuiTheme({
+  typography: {
+    fontFamily: "Lato",
+  },
+});
 
 const NavBar = (props) => {
+  const classes = useStyles();
   return (
-    <header className={styles.navbar}>
-      <nav className={styles.navbar_navigation}>
-        <div className={styles.navbar_toggle_button}>
-          <DrawerToggleButton click={props.drawerClickHandler} />
-        </div>
-        <div className={styles.navbar_logo}>
-          <img
-            src="/images/SocietyLogo.png"
-            alt="society logo"
-            className={styles.image}
-          />
-        </div>
-        <div className={styles.spacer}></div>
-        <div className={styles.navbar_items}>
-          <ul id="nav-mobile" className="center">
-            <li>
-              <Link href="/">Home</Link>
-            </li>
-            <li>
-              <Link href="/events">Events</Link>
-            </li>
+    <ThemeProvider theme={theme}>
```

Comment:
Hmm actually do you really need a Theme Provider for this? Theme is usually only used for project-wide settings